kube-client/src/client/auth/oidc.rs

```diff
@@ -137,11 +131,11 @@ pub mod errors {
     }
 }
 
-const BASE64_ENGINE: FastPortable = FastPortable::from(
+const BASE64_ENGINE: engine::GeneralPurpose = engine::GeneralPurpose::new(
```

Comment:
```suggestion
const JWT_BASE64_ENGINE: engine::GeneralPurpose = engine::GeneralPurpose::new(
```

Could be good to clarify _why_ we use a custom engine.